api/grpc_disruption_test.go

```diff
@@ -0,0 +1,345 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package api_test
+
+import (
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("GRPCDisruption Validation", func() {
```

Comment:
Admittedly I dont know how ginkgo and gomega work very well, but do we not need any `Context` blocks?

Reply:
Okay, I did some reading and it _*appears*_ that `Describe` and `Context` are just two mechanisms of grouping tests (`Context` is a nested scope in the case that you want to test different types of outcomes for one topic).

`By` is also a way of nesting multiple behaviors into one `It`.